### ISSUE TITLE
Required fields missing when returning composition

### ISSUE THREAD

**Issue by nr23730** [ehrbase/ehrbase#218]
The returned representation of compositions misses some attributes:
- /rm_version
- //[CareEntry]/item_tree/name
- //[CareEntry]/language
- //[CareEntry]/encoding

Reproducing is simple: Upload a composition with `PREFER: return=representation` take the response and try to upload it again.

You will get an error similar to this one:
```
java.lang.IllegalArgumentException: Message at /name (/content[openEHR-EHR-OBSERVATION.laboratory_test_result.v1, 1]/data[at0001]/events[at0002, 1]/data[at0003]/name):  Attribute name of class ITEM_TREE does not match existence 1..1
Message at /language (/content[openEHR-EHR-OBSERVATION.laboratory_test_result.v1, 1]/language):  Attribute language of class OBSERVATION does not match existence 1..1
Message at /encoding (/content[openEHR-EHR-OBSERVATION.laboratory_test_result.v1, 1]/encoding):  Attribute encoding of class OBSERVATION does not match existence 1..1
Message at /rm_version (/archetype_details/rm_version):  Attribute rm_version of class ARCHETYPED does not match existence 1..1
```

As I had issues like this one multiple times before it might be a good idea to add a robot test to check if the output is consistent to the required input. :)

**Comment by chevalleyc** [comment]
These are addressed with the following PRs:
https://github.com/ehrbase/ehrbase/pull/246
https://github.com/ehrbase/ehrbase/pull/247

**Comment by nr23730** [comment]
This issue is not solved until #275 is merged!
